test/Microsoft.Framework.Runtime.Tests/VersionUtilityFacts.cs

```diff
@@ -7,48 +7,55 @@ namespace Microsoft.Framework.Runtime.Tests
     public class VersionUtilityFacts
     {
         [Theory]
-        [InlineData("net45", "dnx451", false)]
+        [InlineData("net45", "aspnet50", false)]
```

Comment:
This is restoring the old test values. Eventually they can be dnxified but for now it's sufficient to add the below cases